src/replica/duplication/load_from_private_log.cpp

```diff
@@ -168,6 +171,8 @@ void load_from_private_log::find_log_file_to_start()
 void load_from_private_log::find_log_file_to_start(
     const mutation_log::log_file_map_by_index &log_file_map)
 {
+    auto cleanup = dsn::defer([this]() { _duplicator->set_duplication_plog_checking(false); });
```

Comment:
Is this line necessary ? `_duplicator->set_duplication_plog_checking(false);` would be called twice ?